### PR TITLE
fix autoPhasing 

### PR DESCRIPTION
**Pull request by jobo322**
add an option to change the noise level

### CODE REVIEW

File: src/reim/autoPhaseCorrection.js [hunk 1]
```diff
@@ -74,8 +69,8 @@ export function autoPhaseCorrection(data, options = {}) {
 function autoPhaseRegion(re, im, x0) {
   let start = -180;
   let stop = 180;
-  let nSteps = 20;
-  let maxSteps = 3;
+  let nSteps = 2;
+  let maxSteps = 13;
```
Comment (lpatiny) [inline src/reim/autoPhaseCorrection.js:73]: Should not those 2 parameters be options if you need to change them from time to time ?

File: src/reim/autoPhaseCorrection.js [hunk 2]
```diff
@@ -7,23 +7,19 @@ import { phaseCorrection } from './phaseCorrection';
  */
```
Comment (lpatiny) [inline src/reim/autoPhaseCorrection.js:7]: Could you add the documentation for the options ?

**Comment by lpatiny** [comment]
Please also check eslint. Before push you should do a `npm run eslint-fix` and fix issues if they are still some left.

File: src/reim/autoPhaseCorrection.js (hunk 1 above)
Comment (jobo322) [reply src/reim/autoPhaseCorrection.js]: I am just looking for a better sampling

File: src/reim/autoPhaseCorrection.js (hunk 2 above)
Comment (jobo322) [reply src/reim/autoPhaseCorrection.js:14]: It's done

**Comment by jobo322** [comment]
@lpatiny I did add getNoiseLevel in X because of it has just one array as input, Is it ok?

**Pull request merged**